src/test/java/delete_task_use_case/DeleteTaskControllerTest.java

```diff
@@ -0,0 +1,21 @@
+package delete_task_use_case;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class DeleteTaskControllerTest {
+    private final static DeleteTaskPresenter outputBoundary = new DeleteTaskPresenter();
+    private final static DeleteTaskDataAccess dsGateway = new DeleteTaskDataAccess();
+    private final static int dayIndex = 5;
+    private final static String taskTitle = "Task";
+
+    @Test
+    public void testDeleteTaskController() {
```

Comment:
You should explicitly check whether the days instance attribute of your DataAccess class is mutated by initializing it with a Day and a Task, and then deleting the task.

Reply:
Done, as suggested.